lib/bucky/test_equipment/verifications/status_checker.rb

```diff
@@ -178,6 +180,17 @@ def format_href(base_url, href)
           href.insert(0, '/') unless href.match(%r{^/|^#})
           base_url + href
         end
+
+        def redirect_and_http_status_check(response, http_status_check_args)
```

Comment:
I think we need error handling of redirect loop.(we don't have to implement on this PR)

Reply:
We can think about it in the next improvement.